src/pyodide/python-entrypoint-helper.js

```diff
@@ -176,7 +182,7 @@ async function preparePython() {
 function makeHandler(pyHandlerName) {
   return async function (...args) {
     try {
-      const mainModule = await preparePython();
+      const mainModule = await enterJaegerSpan("prep_python", async () => await preparePython());
```

Comment:
Should break this down into a span for `getPyodide` and a span for `setupPackages`.

Reply:
We can have sub-spans, so I'll just add spans inside `preparePython` too.